packages/components/src/DataList/components/DataListFilters/components/DataListSort/DataListSort.test.tsx

```diff
@@ -81,7 +81,7 @@ describe("DataListSort", () => {
     });
 
     it("should close the popover when the button is clicked again", async () => {
-      await userEvent.click(screen.getByRole("combobox"));
+      await userEvent.click(screen.getByTestId("ATL-Combobox-Overlay"));
```

Comment:
I'm fixing this quickly but I'm not sure how I feel about this set of tests in DataListSort. it's testing Combobox behavior that is already tested elsewhere.